qtop.py

```diff
@@ -1585,7 +1591,7 @@ def get_document(scheduling_system):
                         from xml.etree import ElementTree as etree
 
                 if options.TRANSPOSE:
-                    config['transpose_wn_matrices'] = 'False' if config['transpose_wn_matrices'] == 'True' else 'True'
+                    config['transpose_wn_matrices'] = not config['transpose_wn_matrices']
```

Comment:
very good thing to be done with the multiple evals; 

generally, any type of code repetition should be subjected to scrutiny, no matter how small!